docs/sdkx_android/identity-users.mdx

```diff
@@ -0,0 +1,256 @@
+---
+sidebar_position: 45
+title: Identity User Management
+description: Learn more about identifying and managing users.
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  Image,
+} from "@site/src/components/forDocs";
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [Helpshift installWithPlatformId API](/sdkx_android/getting-started#start-using)
+
+</Admonition>
+
+## Managing users with the Identity System​
+
+Setting up the Identity System requires some development effort, but provides many advantages over the legacy method in the form of faster context collection, easier agent experience, enhanced security and spam protection.
+
+### Prerequisites​
+
+<Admonition type="info" title="NOTE">
+  This feature is in closed beta. Contact Helpshift Customer Success team to
```

Comment:
Consider linking the KB article here to give more details about the feature to the reader. If the KB article is already added somewhere prominently, you can ignore this comment.

Reply:
yes, KB article is added in other part docs